src/api.ts

```diff
@@ -5,6 +5,16 @@ export type Talent = {
   firstName: string;
   lastName: string;
   courseId: string;
+  capstoneProject: CapstoneProject;
+};
+
+export type CapstoneProject = {
+  desktop: boolean;
```

Comment:
The naming could indicate that this is a boolean. wdyt about
```suggestion
  isDesktop: boolean;
```

Reply:
Great addition, changed 🤖